sdk/web-pubsub/web-pubsub-express/review/web-pubsub-express.api.md

```diff
@@ -62,7 +62,11 @@ export interface DisconnectedRequest {
 export type UserEventRequest = {
     context: ConnectionContext;
     data: string;
-    dataType: "text" | "json";
+    dataType: "text";
+} | {
+    context: ConnectionContext;
+    data: any;
```

Comment:
I'd prefer `unknown`, since strictly speaking, it can't really be `any` as JSON only supports a subset of types.